pkg/imgpkg/registry/keychain_test.go

```diff
@@ -1,760 +0,0 @@
-// Copyright 2021 VMware, Inc.
```

Comment:
This test is basically just testing the libraries and the ordering of the keychain. It'd be nice to have some automated tests around this behavior but this is the wrong level to test at